### PR TITLE
[MRG] nicer phantom tutorials

### PR DESCRIPTION
**Pull request by agramfort**
cc @yousrabk

### CODE REVIEW

File: tutorials/plot_brainstorm_phantom_elekta.py [hunk 1]
```diff
@@ -99,7 +109,18 @@
...
+fig, (ax1, ax2) = plt.subplots(nrows=2, ncols=1)
 diffs = 1000 * np.sqrt(np.sum((dip.pos - actual_pos) ** 2, axis=-1))
-print('Differences (mm):\n%s' % diffs[:, np.newaxis])
-print('μ = %s' % (np.mean(diffs),))
+print('mean(position error) = %s' % (np.mean(diffs),))
+ax1.bar(event_id, diffs)
+ax1.set_xlabel('Dipole index')
+ax1.set_ylabel('Loc. error (mm)')
+
+angles = np.arccos(np.abs(np.sum(dip.ori * actual_ori, axis=1)))
```
Comment (larsoner) [inline tutorials/plot_brainstorm_phantom_elekta.py]: why not `np.dot(dip.ori, actual_ori)`?

File: tutorials/plot_brainstorm_phantom_elekta.py [hunk 2]
```diff
@@ -99,7 +109,18 @@
...
+print('mean(angle error) = %s' % (np.mean(angles),))
+ax2.bar(event_id, angles)
+ax2.set_xlabel('Dipole index')
+ax2.set_ylabel('Angle error (rad)')
```
Comment (larsoner) [inline tutorials/plot_brainstorm_phantom_elekta.py]: If you want, you could also do the amplitude error

File: tutorials/plot_brainstorm_phantom_elekta.py (hunk 1 above)
Comment (agramfort) [reply tutorials/plot_brainstorm_phantom_elekta.py]: it's more pedantic hence better :)

**Comment by larsoner** [comment]
LGTM +1 for merge

https://5826-1301584-gh.circle-artifacts.com/0/tmp/circle-artifacts.TScYaXo/html/auto_tutorials/plot_brainstorm_phantom_elekta.html

**Comment by larsoner** [comment]
@yousrabk can you look?

**Comment by yousrabk** [comment]
Looks good to me @agramfort @larsoner

**Comment by agramfort** [comment]
ok I just made the fig smaller following @yousrabk feedback

merging

thanks for the review

**Pull request merged**